generator/internal/genclient/parser/openapi/openapi.go

```diff
@@ -269,7 +265,7 @@ func makePathTemplate(template string) []genclient.PathSegment {
 	return segments
 }
 
-func makeRequestMessage(api *genclient.API, operation *v3.Operation, template string) (*genclient.Message, error) {
+func makeRequestMessage(api *genclient.API, operation *v3.Operation, template string) (*genclient.Message, string, error) {
```

Comment:
nit: could you add a comment to this function as it is no longer clear what is being returned by reading the signature.

Reply:
Done.